pkg/parsers/starknet/starknet.go

```diff
@@ -22,16 +22,22 @@ type EntryPointByType struct {
 }
 
 type Arg struct {
-	GenericID string `json:"generic_id"`
+	GenericID      string         `json:"generic_id"`
+	Size           int            `json:"size"`
+	DebugName      string         `json:"debug_name"`
+	PanicInnerType PanicInnerType `json:"panic_inner_type"`
+}
+
+type PanicInnerType struct {
```

Comment:
Can we leave some kind of comment about this?

Reply:
yep, comment left